week-4/E-arrays-of-objects/exercise-1.js

```diff
@@ -19,17 +19,24 @@ var person3 = {
     age: 20
 };
 
-/* 
+/* {
 DO NOT EDIT ANYTHING ABOVE THIS LINE
 WRITE YOUR CODE BELOW
 */
+function isName(person){
+    return person.name;
+}
+
+function isUnder28(person){
+    if(person.age < 28){
+     return person;   
```

Comment:
I know when you executed this it worked... to the dismay of most coders JavaScript often allows you to do things that weren't meant to work such as in this case. By definition, the callback used for `filter()` needs to return a boolean value: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter.